Apps/Playground/Win32/App.cpp

```diff
@@ -109,7 +109,7 @@ namespace
                 OutputDebugStringA(message);
             });
 
-            Babylon::Polyfills::Window::Initialize(env);
+            Babylon::Polyfills::Window::Initialize(env, (void*)&hWnd);
```

Comment:
>hWnd [](start = 64, length = 4)

how does this work when hand in a pointer to a swapchainpanel comapred to a window?

[https://github.com/BabylonJS/BabylonReactNative/blob/95ecb8305edbe58f73f3a49e743d8b7ea428f8c5/Modules/%40babylonjs/react-native/windows/BabylonReactNative/EngineView.cpp#L35](https://github.com/BabylonJS/BabylonReactNative/blob/95ecb8305edbe58f73f3a49e743d8b7ea428f8c5/Modules/%40babylonjs/react-native/windows/BabylonReactNative/EngineView.cpp#L35)

Reply:
Good question, I'll have to look into how the BabylonNative module interacts with the rest of BabylonNative @syntheticmagus, do you know off hand how we initialize the graphics with a swapchainpanel?

I plan to remove this parameter, @ryantrem raises a good point that we can use `Graphics::Impl::GetNativeWindow()` instead of keeping a pointer to the window, we also persist a pointer to the window type as well.

Reply:
Changed in the latest implementation to no longer manage a reference to a `Graphics` object, we now statically retrieve a Graphics::Impl via Javascript to retrieve any Graphics related info.